inc/fields/ldapselectfield.class.php

```diff
@@ -70,6 +70,54 @@ public static function getName() {
       return __('LDAP Select', 'formcreator');
    }
 
+   public function prepareQuestionInputForSave($input) {
+      // Fields are differents for dropdown lists, so we need to replace these values into the good ones
+      if (isset($input['ldap_auth'])
+          && !empty($input['ldap_auth'])) {
+
+         $config_ldap = new AuthLDAP();
+         $config_ldap->getFromDB($input['ldap_auth']);
+
+         if (!empty($input['ldap_attribute'])) {
+            $ldap_dropdown = new RuleRightParameter();
+            $ldap_dropdown->getFromDB($input['ldap_attribute']);
+            $attribute     = array($ldap_dropdown->fields['value']);
+         } else {
+            $attribute     = array();
+         }
+
+         // Set specific error handler to catch LDAP errors
```

Comment:
This function appears in the iff only because I moved its code. OK to rename it here